src/GrooveApiDataContract/ErrorCode.cs

```diff
@@ -123,27 +123,27 @@ public enum ErrorCode
         #endregion
 
         #region AccessToken errors
-        [Description("Azure Marketplace access token required")]
+        [Description("Application access token required")]
```

Comment:
yup

---
In reply to: [94793263](https://github.com/Microsoft/groove-api-sdk-csharp/pull/1#discussion_r94793263) [](ancestors = 94793263)